README.md

```diff
@@ -687,36 +688,53 @@ npx turbo --help
 
 #### Commands
 
-##### `crypto-fund`
+##### `balance`
 
-Fund a wallet with Turbo Credits by submitting a payment transaction for the crypto amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for top up processing.
+Get the balance of a wallet or native address in Turbo Credits.
```

Comment:
Is there anywhere here (or in linked docs) we disambiguate wallet address and native address?

Reply:
I don't think so. I'd say those would be the same thing to a user, they'd expect wallet address to be their native address. The one thats different could be perceived arweave b64 encoded address

Reply:
We could link out the docs.ar.io glossary if we want and use the term "normalized address" rather than "wallet address":
https://docs.ar.io/glossary.html#normalized-address
https://docs.ar.io/glossary.html#native-address